File: app/src/main/java/eu/basicairdata/graziano/gpslogger/EventBusMSG.java
/*
 * EventBusMSG - Java Class for Android
 * Created by G.Capelli on 05/08/17.
 * This file is part of BasicAirData GPS Logger
 *
 * Copyright (C) 2011 BasicAirData
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program. If not, see <http://www.gnu.org/licenses/>.
 */

package eu.basicairdata.graziano.gpslogger;

public class EventBusMSG {
    public static final short APP_RESUME = 1;
    public static final short APP_PAUSE = 2;
    public static final short NEW_TRACK = 3;
    public static final short UPDATE_FIX = 4;
    public static final short UPDATE_TRACK = 5;
    public static final short UPDATE_TRACKLIST = 6;
    public static final short UPDATE_SETTINGS = 7;
    public static final short REQUEST_ADD_PLACEMARK = 8;
    public static final short ADD_PLACEMARK = 9;
    public static final short APPLY_SETTINGS = 10;
    public static final short TOAST_TRACK_EXPORTED = 11;
    public static final short UPDATE_JOB_PROGRESS = 13;
    public static final short NOTIFY_TRACKS_DELETED = 14;
    public static final short UPDATE_ACTIONBAR = 15;
    public static final short REFRESH_TRACKLIST = 16;
    public static final short REFRESH_TRACKTYPE = 17;

    public static final short TRACKLIST_DESELECT = 24;
    public static final short TRACKLIST_SELECT = 25;
    public static final short INTENT_SEND = 26;
    public static final short TOAST_UNABLE_TO_WRITE_THE_FILE = 27;

    public static final short ACTION_BULK_DELETE_TRACKS = 40;
    public static final short ACTION_BULK_EXPORT_TRACKS = 41;
    public static final short ACTION_BULK_VIEW_TRACKS = 42;
    public static final short ACTION_BULK_SHARE_TRACKS = 43;
    public static final short TRACKLIST_RANGE_SELECTION = 44;
    public static final short ACTION_EDIT_TRACK = 45;
}

File: app/src/main/kotlin/eu/basicairdata/graziano/gpslogger/EventBusMSG.kt
package eu.basicairdata.graziano.gpslogger

object EventBusMSG {
    const val APP_RESUME: Short = 1
    const val APP_PAUSE: Short = 2
    const val NEW_TRACK: Short = 3
    const val UPDATE_FIX: Short = 4
    const val UPDATE_TRACK: Short = 5
    const val UPDATE_TRACKLIST: Short = 6
    const val UPDATE_SETTINGS: Short = 7
    const val REQUEST_ADD_PLACEMARK: Short = 8
    const val ADD_PLACEMARK: Short = 9
    const val APPLY_SETTINGS: Short = 10
    const val TOAST_TRACK_EXPORTED: Short = 11
    const val UPDATE_JOB_PROGRESS: Short = 13
    const val NOTIFY_TRACKS_DELETED: Short = 14
    const val UPDATE_ACTIONBAR: Short = 15
    const val REFRESH_TRACKLIST: Short = 16
    const val REFRESH_TRACKTYPE: Short = 17

    const val TRACKLIST_DESELECT: Short = 24
    const val TRACKLIST_SELECT: Short = 25
    const val INTENT_SEND: Short = 26
    const val TOAST_UNABLE_TO_WRITE_THE_FILE: Short = 27

    const val ACTION_BULK_DELETE_TRACKS: Short = 40
    const val ACTION_BULK_EXPORT_TRACKS: Short = 41
    const val ACTION_BULK_VIEW_TRACKS: Short = 42
    const val ACTION_BULK_SHARE_TRACKS: Short = 43
    const val TRACKLIST_RANGE_SELECTION: Short = 44
    const val ACTION_EDIT_TRACK: Short = 45
}


